jni/include/commons.h

```diff
@@ -26,12 +26,34 @@ namespace knn_jni {
          */
         jlong storeVectorData(knn_jni::JNIUtilInterface *, JNIEnv *, jlong , jobjectArray, jlong);
 
+        /**
```

Comment:
Should we specify the data type the bytes will be stored in?

Reply:
It is in the parameter definition.
`@param data 2D byte array containing data to be stored in native memory.`